monkestation/code/modules/ranching/chickens/_chicken.dm

```diff
@@ -0,0 +1,458 @@
+	var/list/friends = list()
+
+/mob/living/simple_animal/chick/Initialize(mapload)
+	. = ..()
+	pixel_x = rand(-6, 6)
+	pixel_y = rand(0, 10)
+	GLOB.total_chickens++
+
+/mob/living/simple_animal/chick/proc/assign_chick_icon(mob/living/simple_animal/chicken/chicken_type)
+	if(!chicken_type) // do we have a grown type?
+		return
+
+	var/mob/living/simple_animal/chicken/hatched_type = new chicken_type
+	icon_state = "chick_[hatched_type.icon_suffix]"
+	held_state = "chick_[hatched_type.icon_suffix]"
+	icon_living = "chick_[hatched_type.icon_suffix]"
+	icon_dead = "dead_state" //TODO: add dead sprites for each chick / chicken
+	qdel(hatched_type)
+
+/mob/living/simple_animal/chick/Life()
+	. =..()
+	if(!.)
+		return
+	if(!stat && !ckey)
+		amount_grown += rand(1,2)
+		if(amount_grown >= 100)
+			var/mob/living/simple_animal/chicken/new_chicken = new grown_type(src.loc)
+			new_chicken.Friends = src.friends
+			new_chicken.age += rand(1,10) //add a bit of age to each chicken causing staggered deaths
+			if(istype(new_chicken, /mob/living/simple_animal/chicken/glass))
+				for(var/list_item in glass_egg_reagent)
+					new_chicken.glass_egg_reagents.Add(list_item)
+
+			if(istype(new_chicken, /mob/living/simple_animal/chicken/stone))
+				if(production_type)
+					new_chicken.production_type = production_type
+			qdel(src)
+
+/mob/living/simple_animal/chick/death(gibbed)
+	friends = null
+	GLOB.total_chickens--
+	..()
+
+/mob/living/simple_animal/chick/Destroy()
+	friends = null
+	if(stat != DEAD)
+		GLOB.total_chickens--
+	return ..()
+
+/mob/living/simple_animal/chick/holo/Life()
+	..()
+	amount_grown = 0
+
+/mob/living/simple_animal/chicken
+	name = "\improper chicken"
+	desc = "Hopefully the eggs are good this season."
+	gender = FEMALE
+
+	maxHealth = 15
+	mob_biotypes = list(MOB_ORGANIC, MOB_BEAST)
+
+	icon = 'monkestation/icons/mob/ranching/chickens.dmi'
+	icon_state = "chicken_white"
+	icon_living = "chicken_white"
+	icon_dead = "dead_state"
+	head_icon = 'icons/mob/pets_held_large.dmi'
+	held_state = "chicken_white"
+
+	speak_chance = 2
+	speak = list("Cluck!","BWAAAAARK BWAK BWAK BWAK!","Bwaak bwak.")
+	speak_emote = list("clucks","croons")
+	emote_hear = list("clucks.")
+	emote_see = list("pecks at the ground.","flaps its wings viciously.")
+
+	response_help  = "pets"
+	response_disarm = "gently pushes aside"
+	response_harm   = "kicks"
+	attacktext = "kicks"
+
+	density = FALSE
+	turns_per_move = 3
+	butcher_results = list(/obj/item/food/meat/slab/chicken = 2)
+	ventcrawler = VENTCRAWLER_ALWAYS
+	can_be_held = TRUE
+	worn_slot_flags = ITEM_SLOT_HEAD
+	pass_flags = PASSTABLE | PASSMOB
+	mob_size = MOB_SIZE_SMALL
+	chat_color = "#FFDC9B"
+	mobchatspan = "stationengineer"
+
+	do_footstep = TRUE
+
+
+/mob/living/simple_animal/chicken/Initialize(mapload)
+	. = ..()
+	pixel_x = rand(-6, 6)
+	pixel_y = rand(0, 10)
+	GLOB.total_chickens++
+	chicken_type = src
+	assign_chicken_icon()
+
+	if(prob(40))
+		gender = MALE
```

Comment:
There should be a male and female type for the base chicken, so maps have 100% chances of a breeding pair.
This will need a check to make sure it doesn't overwrite

Reply:
fixed in newer commits already with the hen subtype